lib/src/util.dart

```diff
@@ -49,6 +52,12 @@ Iterable<Token> allComments(Token beginToken) sync* {
     }
     currentToken = currentToken.next;
   }
+  // Also check comments preceding EOF.
```

Comment:
Ohh good catch. 

#nit Instead of adding a loop for this case, could we just update the while-loop to remove the `isEof` check, since the EOF token should have a null `.next`?
```diff
-while (currentToken != null && !currentToken.isEof) {
+ hile (currentToken != null) {
```

Reply:
I tried that, but for some reason `.next` of the EOF token is itself so it made an infinite loop 🤷‍♀️

Reply:
Ah, gotcha, weeeird 👍